project/requirements.txt

```diff
@@ -1,5 +1,5 @@
 pyyaml
-pydantic
+pydantic < 2.0
```

Comment:
Of course it would, be better to upgrade (I already started this in another branch, but I delayed this, since I would like to get it running first). Without the limitation errors are thrown, since packages have been restructured in version 2.0